src/docs/sphinx/requirements.txt

```diff
@@ -8,4 +8,3 @@ pandas
 # using plantuml for diagrams
 sphinxcontrib-plantuml
 sphinx-argparse
-pydata-sphinx-theme
```

Comment:
Do you know who added this package and why?
We should checking that the feature it was allowing still works.

Reply:
so, can this be merged?

Reply:
> so, can this be merged?

I think so. 
This change should only affect sphinx on ReadTheDocs, and not sphinx on LC/local sphinx builds where pydata-sphinx-theme was inteded for use.